pkg/cluster/bootstrap.go

```diff
@@ -309,17 +309,46 @@ func migrateBootstrapData(ctx context.Context, data io.Reader, files bootstrap.P
 
 const systemTimeSkew = int64(3)
 
+// isMigrated checks to see if the given bootstrap data
+// is in the latest format.
+func isMigrated(buf io.ReadSeeker) bool {
```

Comment:
Should we always seek 0,0 when we start invoking this function?

Reply:
Yeah, not a bad idea. I will test the change I just made and push up for reapproval.